assets/styles/objects/_grid.scss

```diff
@@ -0,0 +1,167 @@
+// ==========================================================================
+// Grid helper
+// ==========================================================================
+// Help: https://css-tricks.com/snippets/css/complete-guide-grid/
+//
+// How to use :
+/**
+*```html
+* <div class="o-grid -col-4 -col-12@from-medium -gutters">
+*     <div class="o-grid_item u-gc-1/2 u-gc-3/9@from-medium">
+*           <p>Hello</p>
+*     </div>
+*     <div class="o-grid_item u-gc-3/4 u-gc-9/13@from-medium">
+*           <p>Hello</p>
+*     </div>
+* </div>
+* ```
+**/
+
+.o-grid {
+    display: grid;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    list-style: none;
```

Comment:
Maybe use the following for style reset:
```
@at-root ul#{&},
@at-root ol#{&} {
    margin: 0;
    padding: 0;
    list-style: none;
}
```

Reply:
@devenini Is this still relevant?

Reply:
Sorry not too familiar with this sass selector. What is the advantage of using `@at-root` here?

Reply:
Testing out the suggestion, the `@at-root` is quite powerful. I wish I knew about this feature sooner 😆.

> […] causes everything within it to be emitted at the root of the document instead of using the normal nesting. […]

```scss
.o-grid {
    // …

    @at-root ul#{&},
    @at-root ol#{&} {
        margin: 0;
        padding: 0;
        list-style: none;
    }
}
```

```css
.o-grid {
    // …
}

ul.o-grid,
ol.o-grid {
    margin: 0;
    padding: 0;
    list-style: none;
}


.o-grid.-col-12 {
    // …
}

// …
```

Reply:
It allows you to add styles for a class with a specific selector, here `ul.o-grid` and `ol.o-grid` and only reset the list styles when it is actually one